src/stores/BreadcrumbsStore.ts

```diff
@@ -87,6 +88,23 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
 
         this.matrixClient.on("Room.myMembership", this.onMyMembership);
         this.matrixClient.on("Room", this.onRoom);
+
+        const client = MatrixClientPeg.get();
+        const breadcrumbs = client.store.getAccountData("im.vector.setting.breadcrumbs");
```

Comment:
Hmm, it feels a bit strange to go directly to the account data here when the main purpose of this store is reading and tracking the breadcrumbs anyway... 

Maybe it fit a bit better in `updateRooms`, and you could check for the case of transitioning from no rooms to some rooms, so it only fires once...?